README.md

```diff
@@ -3,21 +3,22 @@
 > Backend da NewSchool, desenvolvido em NodeJS
 > Banco de Dados MySQL
 
-
 # Como rodar o projeto
 
 Você pode rodar o projeto direto da sua máquina.
 
-1 - Faça uma cópia do arquivo **.env.example** para **.env**
+1 - Tenha a versão 12 do Node.js instalada em sua máquina.
```

Comment:
Acho melhor colocar as mesmas versões do CI, 8, 11 ou 12

Reply:
Não entendi. Por que não fechar com a última estável?

Reply:
Porque a pessoa pode ter qualquer uma dessas, não é?
E se for usar a última estável é a 13.x. A 12 ta em LTS já

Reply:
Talvez a gente devesse até tirar a 8 e a 11 já que não estão nem em LTS ativo

Reply:
> Não entendi. Por que não fechar com a última estável?

Acho que deveríamos seguir o que o CI está seguindo atualmente. Se a pessoa tiver umas dessas 3 versões, o projeto roda, entendeu? Não é obrigatório ter a última.

Reply:
Vamos fechar na 12.14.0 ? precisamos definir isso o quanto antes.

Reply:
Voto a favor. Bora fechar com essa 12.14.0 (caso seja estável)

Reply:
Acho que seria legal usar o NVM para isso. Algo do tipo:

1. Check the NodeJS version

   This project is using the [NodeJS 12.14.0](https://nodejs.org/en/blog/release/v12.14.0/) version. If you have [NVM]() installed in your machine just execute the follow command

   ```bash
   # If is your first time on this project run
   # nvm install
   # After that you can use
   nvm use
   ```